### ISSUE TITLE
Compiled assets in 'lib' folder are not in the release bundles

### ISSUE THREAD

**Issue by CloudNiner** [azavea/loam#16]
I downloaded the latest release from https://github.com/azavea/loam/archive/1.0.0-alpha.8.tar.gz

It did not include the lib folder minified bundles:
```
andrew@tango ~/Downloads: ls -l loam-1.0.0-alpha.8
total 400
-rw-r--r--@  1 andrew  staff   11343 Apr  6 17:07 LICENSE
-rw-r--r--@  1 andrew  staff    1251 Apr  6 17:07 README.md
-rw-r--r--@  1 andrew  staff    1753 Apr  6 17:07 karma.conf.js
-rw-r--r--@  1 andrew  staff    1616 Apr  6 17:07 package.json
drwxr-xr-x@ 10 andrew  staff     320 Apr  6 17:07 src
drwxr-xr-x@  4 andrew  staff     128 Apr  6 17:07 test
-rw-r--r--@  1 andrew  staff    1186 Apr  6 17:07 webpack.config.js
-rw-r--r--@  1 andrew  staff  172960 Apr  6 17:07 yarn.lock
```

Instead as a workaround, I cloned the repo, then ran:
```
yarn install --pure-lockfile
yarn build
```
and then copied the contents of the `lib` folder to my project.

**Comment by ddohler** [comment]
Okay, this turned out to be a combination of a couple things:
- GitHub is autogenerating releases based on tags, but the `lib` folder is in `.gitignore` so it doesn't get included in the autogenerated bundles.
- I botched the `alpha-8` release to NPM and then forgot to push the new `alpha-9` tag so GitHub didn't pick it up.

I think for now, probably the best way to do this is for you to use `npm` / `yarn` to install locally; the NPM bundles do have the `lib` folder.

I've also opened an issue to add deployment functionality to Travis so that merges to master get automatically bundled and pushed up to both GitHub releases and NPM. That should avoid both botched deploys and the misleading GitHub autogenerated releases. I've also opened an issue to add installation instructions to the README; it wasn't clear that this package is available on NPM in the first place.
